### ISSUE TITLE
broken content-type serving files in sub directories

### ISSUE THREAD

**Issue by retog** [herudi/static-files#3]
I'm using static-files with oak and I'm getting broken content-type headers for files in sub-directories: 

```
$ curl -D - 'http://localhost:8000/test.js' 
HTTP/1.1 200 OK
accept-ranges: bytes
content-type: application/javascript; charset=utf-8
etag: W/"10-1639076608438"
last-modified: Thu, 09 Dec 2021 19:03:28 GMT
content-length: 10
date: Thu, 09 Dec 2021 19:03:42 GMT

alert("h")

$ curl -D - 'http://localhost:8000/js/test.js' 
HTTP/1.1 200 OK
accept-ranges: bytes
content-type: \js/test.js
etag: W/"10-1639076638723"
last-modified: Thu, 09 Dec 2021 19:03:58 GMT
content-length: 10
date: Thu, 09 Dec 2021 19:04:08 GMT

alert("h")
```

**Comment by herudi** [comment]
thanks for creating this issue. I will investigate your case first

**Comment by herudi** [comment]
@retog fixes in new version 1.1.3
